addons/mass_mailing/models/mailing.py

```diff
@@ -350,6 +351,16 @@ def _compute_next_departure(self):
             else:
                 mass_mailing.next_departure = fields.datetime.now()
 
+    @api.depends('next_departure', 'state')
```

Comment:
Actually it does not depend on state. being in the past just depends on next_departure; the state is just something allowing to know if we care about it or not. In view normally it is already in part controller with state I think, when looking quickly, so it should be good normally.